lib/badfruit/Movies/movie.rb

```diff
@@ -26,6 +26,11 @@ def full_cast
       return @badfruit.parse_actors_array(JSON.parse(@badfruit.get_movie_info(@id, "cast")))
     end
 
+    # Returns director, since RottenTomatoes API is broken (abridged_director doesn't show up in search_by_name results for movies.)
+    def director
+      return Director.new(JSON.parse(@badfruit.get_movie_info(@id, "details"))["abridged_directors"][0])
```

Comment:
Thanks for the reply, quick question:  by caching on the object do you mean storing it as an attribute? I can change it to do that instead.

I've noticed the Rotten Tomatoes API seems to hide some movie information whenever it returns a list of movies. The genres array also doesn't show up in the response for methods like `search_by_name`. I wonder if this is intended or not.